recipes/perl-xml-libxml/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: perl-xml-libxml
-  version: "2.0124"
+  version: "2.0132"
 
 source:
-  url: https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXML-2.0124.tar.gz
-  md5: 30436b85454fba3ae5f6494df598e65c
+  url: https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXML-2.0132.tar.gz
+  md5: 43546fd9a3974f19323f9fb04861ece9
 
 build:
   number: 1
```

Comment:
can we reset this here to 0 - you can use [ci skip] if you do it - it should not influence the previous test